bookshelf/pom.xml

```diff
@@ -95,6 +95,24 @@ Copyright 2015 Google Inc. All Rights Reserved.
       </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
```

Comment:
When I tried this in java-docs-samples the maven-checkstyle-plugin was not getting run on submodules. Maybe there is a way to do it, but I verified it was not running by introducing style errors intentionally and not seeing an error when running `mvn verify` on the root plugin.